aiohttp/test_utils.py

```diff
@@ -430,12 +430,12 @@ def get_app(self) -> Application:
         raise RuntimeError("Did you forget to define get_application()?")
 
     def setUp(self) -> None:
-        try:
-            self.loop = asyncio.get_running_loop()
-        except RuntimeError:
-            self.loop = asyncio.get_event_loop_policy().get_event_loop()
```

Comment:
This bit needed to be recovered in aiohttp v3.8 to preserve the compatibility with Python 3.6: https://github.com/aio-libs/aiohttp/commit/ef6a373b3273eda4b9945ca82491671a1e7bad9f.